app/components/badge-github-actions.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+  tagName: 'span',
+  classNames: ['badge'],
+  repository: alias('badge.attributes.repository'),
+  workflow: computed('badge.attributes.workflow', function() {
+    return this.get('badge.attributes.workflow').replace(' ', '%20');
```

Comment:
This will only replace the first space – it’d have to be `/ /g`. But I agree that this is strange and something like `encodeURIComponent()` would seem to make more sense.